### DISCUSSION TITLE
.md files not rendering html elements

### DISCUSSION THREAD

**Discussion by orazdow** [mdx-js/mdx#2026] [Q&A] [answered]
The following code will only render 'hello' if I use an mdx file. 

	# ok...

	<div>hello</div>

I set the options below in webpack:

	{
		test: /\.mdx?$/,
		use: {
			loader: '@mdx-js/loader',
			options: {
				mdExtensions: ['.md'],
				format: 'md',
				remarkRehypeOptions:{
					allowDangerousHtml: true
				}
			}
		}
	}

but the only way the html will render in a .md file is if I change the options to `format: 'mdx'`, treating the file as mdx. Shouldn't the regular markdown display the html section?

**Comment by wooorm** [comment, marked as answer]
Should work w/ `rehype-raw` (w/ `passThrough` if you have both markdown and mdx files): https://github.com/orgs/mdx-js/discussions/2023#discussioncomment-2649772
